priv/www/js/tmpl/policies.ejs

```diff
@@ -27,7 +27,11 @@
 <% if (vhosts_interesting) { %>
      <td><%= fmt_string(policy.vhost) %></td>
 <% } %>
+<% if (user_policymaker) { %>
```

Comment:
Please add the missing `is`. I do realise that `vhosts_interesting` and probably a few other places have the same issue but we shouldn't make it worse.